_includes/accessibility-tests/wcag-details.html

```diff
@@ -17,6 +17,16 @@
   {% assign criterion_name = "Meaningful sequence" %}
   {% assign criterion_url = "https://www.w3.org/TR/WCAG21/#meaningful-sequence" %}
 {% endif %}
+{% if item.wcag_criterion == "1.3.3" %}
+  {% assign criterion_level = "A" %}
+  {% assign criterion_name = "Meaningful sequence" %}
```

Comment:
This is noted on wcag page as "Sensory characteristics" not "Meaningful sequence" - should it be corrected to reflect what's on WCAG page?
reference: https://www.w3.org/TR/WCAG21/#sensory-characteristics

Reply:
Resolved in [d53cec6](https://github.com/uswds/uswds-site/pull/2697/commits/d53cec6e2edbe1c727df1d0c566c6d6882019cb9)